packages/api/src/controllers/user.ts

```diff
@@ -1525,28 +1528,62 @@ app.post(
       { limit },
     );
 
-    const results = [];
+    const results = {
+      migrated: [],
+      flagged: [],
+    };
+
+    const OLD_USER_CUTOFF = Date.now() - ms("12 months");
```

Comment:
Can you make this a constant in the file, not in the HTTP handler, so we're not constantly calculating this date? Also, can you make this a well-defined fixed date instead of a computation that depends on `Date.now()`? e.g. Just pick the timestamp corresponding to 2023/06/26 and hardcode that in the constant.

Reply:
Actually, making this a fixed value is way more important than moving to a constant in the file. That's a required change otherwise the behavior of this migration API is not deterministic.

All other comments optional.

Reply:
Sure, makes sense, harcoding the millis unix timestamp for that date